src/components/LoginForm/index.js

```diff
@@ -54,27 +54,31 @@ const LoginForm = ({ onSubmit }) => {
   return (
     <>
       <Input
-        label={strings.SIGN_IN.email}
+        label={strings.SIGN_IN.email.toUpperCase()}
         keyboardType="email-address"
         autoCapitalize="none"
         testID="email-input"
         {...inputProps(FIELDS.email)}
       />
       <Input
-        label={strings.SIGN_IN.password}
+        label={strings.SIGN_IN.password.toUpperCase()}
```

Comment:
WDYT about using the upperCase method outside the render component to avoid multiple calls to this method on every render.

IE:

```
const PASSWORD_LABEL = strings.SIGN_IN.password.toUpperCase()

const LoginForm = ({ ... }) => {}
```